test/chromium/tracing.jest.js

```diff
@@ -16,23 +16,18 @@
 
 const fs = require('fs');
 const path = require('path');
-const {FFOX, CHROMIUM, WEBKIT, OUTPUT_DIR, CHANNEL} = require('../utils').testOptions(browserType);
+const {FFOX, CHROMIUM, WEBKIT, OUTPUT_DIR, CHANNEL} = testOptions;
 
-describe('Chromium.startTracing', function() {
-  beforeEach(async function(state) {
-    state.outputFile = path.join(OUTPUT_DIR, `trace-${state.parallelIndex}.json`);
-    state.browser = await state.browserType.launch(state.defaultBrowserOptions);
-    state.page = await state.browser.newPage();
-  });
-  afterEach(async function(state) {
-    await state.browser.close();
-    state.browser = null;
-    state.page = null;
-    if (fs.existsSync(state.outputFile)) {
-      fs.unlinkSync(state.outputFile);
-      state.outputFile = null;
-    }
-  });
+registerFixture('outputFile', async ({parallelIndex}, test) => {
```

Comment:
what happens when I register a fixture in a file? Can it be used in otherfiles? Why not just use beforeEach?

Reply:
beforeEach can't add into the state...